windows/web/fix_ch340.ps1

```diff
@@ -0,0 +1,151 @@
+# AFAIK this is the only HWID that counterfeit CH340s use
+# Well, they have 2 HWIDs according to Windows, but only 1 is needed for detection
+$HardwareID = "USB\VID_1A86&PID_7523"
+
+# ! Currently it gets the driver from my own CDN! This might be a security problem.
+$DriverZipUrl = 'cdn.kouno.xyz/YZSvN1b5.zip'
```

Comment:
i agree we need to change this, I suggest uploading it to Github and use their CDN

Reply:
Or just use a bundled file lol